alembic/versions/e29e1b255d90_create_idx_proj_task_result_warm_cache.py

```diff
@@ -0,0 +1,26 @@
+"""create indexes project task result for warm_cache
+
+Revision ID: e29e1b255d90
+Revises: 2edf951cc6ae
+Create Date: 2018-09-24 11:40:56.979019
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e29e1b255d90'
+down_revision = '2edf951cc6ae'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_index('result_project_id_info_idx', 'result', ['project_id', 'info'])
+    op.create_index('task_project_id_state_idx', 'task', ['project_id', 'state'])
+    op.create_index('project_featured_idx', 'project', ['featured'])
```

Comment:
where is this used?

Reply:
to compute n_featured under pybossa.cache.projects and called during warm up